src/libs/SidebarUtils.ts

```diff
@@ -233,14 +249,23 @@ type ActorDetails = {
 /**
  * Gets all the data necessary for rendering an OptionRowLHN component
  */
-function getOptionData(
-    report: Report,
-    reportActions: Record<string, ReportAction>,
-    personalDetails: Record<number, PersonalDetails>,
-    preferredLocale: ValueOf<typeof CONST.LOCALES>,
-    policy: Policy,
-    parentReportAction: ReportAction,
-): ReportUtils.OptionData | undefined {
+function getOptionData({
```

Comment:
Same comment here, if there's no benefit to using a single object param, let's go back to individual params

Reply:
Here there are eight params. I think there's a significant benefit to having them in an object.